fixtures/parcel-transformer-test-extract-app/.parcelrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "@parcel/config-default",
+  "transformers": {
+    "*.js": ["@compiled/parcel-transformer", "..."]
+  },
+  "resolvers": ["@compiled/parcel-resolver-noop", "..."]
```

Comment:
I wanted to keep the parcel plugin tests independent so that issues in plugins don't leak between tests. It does add a lot of fixtures but I did prefer that only the relevant tests were failing when I was working on this branch